tests/virt_autotest/sev_es_guest_verification.pm

```diff
@@ -0,0 +1,191 @@
+# VIRTUAL MACHINE AMD SEV/SEV-ES FEATURES VERIFICATION MODULE
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: This module tests whether SEV/SEV-ES virtual machine has
+# been successfully installed on SEV/SEV-ES enabled physical host by
+# checking SEV/SEV-ES support status on physical host in the first
+# place and then virtual machine itself.
+#
+# Maintainer: Wayne Chen <wchen@suse.com>
+package sev_es_guest_verification;
+
+use base 'virt_feature_test_base';
+use strict;
+use warnings;
+use POSIX 'strftime';
+use File::Basename;
+use testapi;
+use IPC::Run;
+use utils;
+use virt_utils;
+use virt_autotest::common;
+use virt_autotest::utils;
+use version_utils qw(is_sle is_tumbleweed);
+use Utils::Architectures;
+
+sub run_test {
+    my $self = shift;
+
+    $self->check_sev_es_on_host;
+    foreach (keys %virt_autotest::common::guests) {
+        virt_autotest::utils::wait_guest_online($_);
+        $self->check_sev_es_on_guest(guest_name => "$_");
+    }
+    return $self;
+}
+
+=head2 check_sev_es_on_host
+
+  check_sev_es_on_host($self)
+
+Check whether AMD SEV or SEV-ES feature is enabled and active on physical host
+under test. It calls check_sev_es_parameter to perform the task. There is no
+additional argument supported by this subroutine. 
+
+=cut
+
+sub check_sev_es_on_host {
+    my $self = shift;
+
+    record_info('Check SEV/SEV-ES support status on host', 'Only 15-SP2+ host supports AMD SEV and only 15-SP4+ supports AMD SEV-ES.');
+    if (is_x86_64) {
+        if (is_sle('>=15-SP4') or is_tumbleweed) {
+            die 'Host does not support AMD SEV or SEV-ES feature although it is a 15-SP4 or newer produdct.' unless ($self->check_sev_es_parameter(params_to_check => 'sev sev_es') == 0);
```

Comment:
Would you like to modify the die msg a little bit to "the corresponding xx feature is active or inactive on the host", just like how you describe check_sev_es_parameter? The msg "does not support" is easier to be confused with "this product does not support the feature".

Reply:
Done